lib/errors.js

```diff
@@ -0,0 +1,74 @@
+class StorageEmpty extends Error {
+  constructor () {
+    super('No Hypercore is stored here')
+    this.code = 'STORAGE_EMPTY'
+  }
+}
+
+class StorageConflict extends Error {
+  constructor () {
+    super('Another Hypercore is stored here')
+    this.code = 'STORAGE_CONFLICT'
+  }
+}
+
+class InvalidSignature extends Error {
```

Comment:
Would be explicit about this being an error from the remote, i.e. `InvalidRemoteSignature` and `InvalidRemoteCapability`

Reply:
Good point